pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -48,6 +46,9 @@ type Multicluster struct {
 	m                     sync.Mutex // protects remoteKubeControllers
 	remoteKubeControllers map[string]*kubeController
 	networksWatcher       mesh.NetworksWatcher
+
+	// fetchCaRoot maps the certName to cert
```

Comment:
I did a search of "certName" in the PR and only find one reference of "certName" on this line. Can you add the definition of "certName" on this line?

Reply:
this map is the same with pilot/pkg/serviceregistry/kube/controller/namespacecontroller.go:L52:
`	getData func() map[string]string`  and the key is defined in 
`fetchData := func() map[string]string {
				return map[string]string{
					constants.CACertNamespaceConfigMapDataName: string(s.ca.GetCAKeyCertBundle().GetRootCertPem()),
				}`
so all are  old codes actually, just reuse them.

Reply:
The "certName" is introduced in this PR. As"certName" is not defined, it should be defined in the PR.

Reply:
no, all are old codes,  see in `pilot/pkg/bootstrap/server.go:L306`

Reply:
The pilot/pkg/bootstrap/server.go:L306 is as follows. I do not see the definition of "certName" in these lines. In this PR, you added the line "// fetchCaRoot maps the certName to cert". What does "certName" mean?

		if s.kubeClient != nil {
			fetchData := func() map[string]string {
				return map[string]string{
					constants.CACertNamespaceConfigMapDataName: string(s.ca.GetCAKeyCertBundle().GetRootCertPem()),
				}
			}